server/BUILD.bazel

```diff
@@ -16,6 +16,7 @@ go_library(
         "//cache/disk:go_default_library",
         "@com_github_bazelbuild_remote_apis//build/bazel/remote/execution/v2:go_default_library",
         "@com_github_bazelbuild_remote_apis//build/bazel/semver:go_default_library",
+        "@com_github_golang_protobuf//jsonpb:go_default_library",
```

Comment:
`bazel run //:gazelle fix` on this commit produces a modified server/BUILD.bazel file for me- can you apply those changes?